packages/cli/src/resolve-module.ts

```diff
@@ -0,0 +1,21 @@
+import type { IFileSystem } from '@file-services/types';
+import { createRequestResolver } from '@file-services/resolve';
+import type { StylableConfig } from '@stylable/core';
+
+export function createDefaultResolveModule(fs: IFileSystem): StylableConfig['resolveModule'] {
```

Comment:
This is a much larger type than the resolver needs/uses. See it's options. There's another, smaller interface.